src/ResponseDataException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace SilverStripe\WebAuthn;
+
+use RuntimeException;
+
+class ResponseDataException extends RuntimeExecption
```

Comment:
I think some doc blocks for these exceptions would be useful, to describe the situations you'd expect them to be thrown in